Tests/UIKitTests/UIButtonExtensionsTests.swift

```diff
@@ -158,6 +158,21 @@ final class UIButtonExtensionsTests: XCTestCase {
         XCTAssertEqual(button.titleEdgeInsets, UIEdgeInsets(top: 0, left: 10, bottom: 0, right: -10))
     }
 
+    func testCenterTextAndImageVertically() {
```

Comment:
Perhaps add a test to ensure that the width of the frame equals the max width of the title or image frames, and that the button's height is the title + image frames + spacing? That's the expected result, isn't it?

Reply:
No, the button must specify a size, it will not change its size automatically by setting the image and title. So such tests may not be successful